sdk/src/Services/EC2/AWSSDK.EC2.NetStandard.csproj

```diff
@@ -52,18 +52,18 @@
   </Choose>
```

Comment:
This file shouldn't be changed, it'll get overwritten by the generator.

Reply:
Agreed. It was a whitespace change. I will exclude it and commit without running a new dry run.

Reply:
Done.